suzieq/poller/controller/utils/inventory_utils.py

```diff
@@ -161,3 +180,35 @@ def get_inventory_config(conf_type: str, inventory: dict) -> dict:
                 configs[name] = conf_obj
 
     return configs
+
+
+def get_sensible_data(input_method: str, ask_message: str = '') -> str:
+    """This function is used by the inventory to specify sensible data
+
+    The valid methods are:
+        - 'plain:' (default): copy the content of input (can be omitted)
+        - 'env:': get the information from an environment variable
+        - 'ask': write the information on the stdin
+
+    Args:
+        input_method (str): string with info for the sensible value
+        ask_message (str): message to prompt for the 'ask' method
+
+    Raises:
+        InventorySourceError: environment variable not found
+
+    Returns:
+        str: sensible data
+    """
+    sens_data = input_method
+    if input_method.startswith('env:'):
+        sens_data = getenv(input_method.split('env:')[1], '')
```

Comment:
Shouldn't we use `strip()` to remove leading and trailing whitespaces? If for example we write `env: VAR` instead of `env:VAR`